sp800-63c/sec7_fal.md

```diff
@@ -23,11 +23,13 @@ This table presents different requirements depending on whether the assertion is
 |2|Bearer assertion, signed by IdP and encrypted to RP|
 |3|Holder of key assertion, signed by IdP and encrypted to RP|
 
-For example, FAL 1 maps to the OpenID Connect Basic Client profile or SAML Web SSO Artifact Binding profile, with no additional features. FAL 2 additionally requires that the OpenID Connect ID Token or SAML Assertion be encrypted to a public key representing the RP in question. FAL 3 requires the presentation of an additional key bound to the assertion (for example, the use of a cryptographic authenticator) along with all requirements of FAL 2. Note that the additional key presented at FAL 3 need not be the same key used by the subscriber to authenticate to the IdP.
+>**MG: Or do we need to say proxy here as well?**
```

Comment:
No, see note above. Proxies act as a different parties in the transaction, so a proxied transaction is a combination of several sub-transactions that each also have their own aspects. We shouldn't call them out separately here.